app/models.py

```diff
@@ -84,8 +84,10 @@ def password(self, password):
     def verify_password(self, password):
         return check_password_hash(self.password_hash, password)
 
-    def generate_confirmation_token(self, expiration=3600):
+    def generate_confirmation_token(self, expiration=9223372036854775807):
         """Generate a confirmation token to email a new user."""
+
+        # Note: This token should never expire.
```

Comment:
I think we should keep the existing flow for now. I agree that an hour seems too short, but giving them the maximum time seems a bit aggressive. How about a week (604800)?